src/schematools/types.py

```diff
@@ -466,14 +482,33 @@ def is_nested_table(self):
         """Indicates if table is an nested table"""
         return self.nested_table
 
+    def model_name(self) -> str:
+        """Returns model name for this table. Including version number, if needed."""
+
+        from schematools.utils import to_snake_case
```

Comment:
does this need a local import?

Reply:
yes, avoiding circular imports. `schematools.utils` imports `schematools.types`

Reply:
bummer. guess this can't befixded.

I completely missed it: `from schematools import MAX_TABLE_NAME_LENGTH, RELATION_INDICATOR, TMP_TABLE_POSTFIX, types`

FYI, there is a ugly hack that might work:

```
from __future__ import annotations
import typing

...

# at bottom of the file:
if typing.TYPE_CHECKING:
    from schematools import types
```

Reply:
Yeah, but given other methods use same local import hack (and it's easier to read) - we'd do with local import here as well.